app/lib/contexts.json

```diff
@@ -0,0 +1,7 @@
+{
+  "stomachAche": {
+    "context": "stomach-ache",
+    "url": "/symptoms/stomach-ache",
+    "text": "Back to information on stomach ache"
```

Comment:
`text` as a key isn't overly informative but i've not changed it because I didn't want to make it too specific until such a time as there are several contexts and there is a better understanding as to what this might become.